requirements/edx/pre.txt

```diff
@@ -4,6 +4,9 @@
 #   * @mollydb - to check licensing
 #   * One of @e0d, @jarv, or @feanil - to check system requirements
 
+# Use a modern setuptools instead of distribute
+setuptools==9.1
```

Comment:
Gotcha. But if we pinned `nltk==2.0.5`, and [updated that for edx-ora2 as well](https://github.com/edx/edx-ora2/pull/686), then we could use setuptools 14.3.1, right? Or are there other reasons why we can't do that?